rsmtool/notebooks/comparison/score_distributions.ipynb

```diff
@@ -8,6 +8,41 @@
    ]
   },
   {
+   "cell_type": "code",
```

Comment:
There may not be any change between old and new most of the time. And, as new/old agreement is not comparable, I did not put the difference information here. Though human human agreement is present in rsmtool report, I thought it will be better to have this information in the comparison report too. I may change to have only one row if they are same, but do not think this needs to be excluded. Please let me know.